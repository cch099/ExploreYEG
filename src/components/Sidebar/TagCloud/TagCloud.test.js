// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import TagCloudX from './TagCloud';

describe('TagCloudX', () => {
  it('renders correctly', () => {
    const props = {
      // copyright: 'copyright'
    };

    const tree = renderer.create(<TagCloudX {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
