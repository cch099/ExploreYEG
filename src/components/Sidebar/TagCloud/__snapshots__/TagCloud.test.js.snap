// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagCloud renders correctly 1`] = `
<div
  className="tagcloud"
>
  tagcloud
</div>
`;
